cmd/ci-secret-bootstrap/main.go

```diff
@@ -267,8 +274,18 @@ func constructSecrets(config []secretConfig, bwClient bitwarden.Client) (map[str
 			keyWG := &sync.WaitGroup{}
```

Comment:
You don't need this any longer, you can just acquire the max amount from the semaphore and use that

Reply:
Updated.